src/ConnectedMode/Resources.resx

```diff
@@ -301,4 +301,7 @@
   <data name="MuteIssuesService_Error_InsufficientPermissions" xml:space="preserve">
     <value>Credentials you have provided do not have enough permission to resolve issues. It requires the permission 'Administer Issues'.</value>
   </data>
+  <data name="ValidateCredentials_Fails" xml:space="preserve">
+    <value>[ConnectedMode] Validating credentials failed</value>
```

Comment:
Credential validation failed + maybe add more info why, as I think there's the error message in the dto

Reply:
Sure. But it would be still nice to see the reason in the logs as well, unless we already receive a log from slcore, in which case you can leave this log line as-is

Reply:
I was missing a logging statement, which I did add it in the latest PR.